### DISCUSSION TITLE
can i add class in my needs

### DISCUSSION THREAD

**Discussion by madfortech** [saadeghi/daisyui#1553] [Q&A] [answered]
Example
<label for="my-modal-4" class="btn">open modal</label>

<input type="checkbox" id="my-modal-4" class="modal-toggle" />
<label for="my-modal-4" class="modal cursor-pointer">
  <label class="modal-box relative" for="">
    **<h3 class="text-lg font-bold">Congratulations random Internet user!</h3>**
    <p class="py-4">You've been selected for a chance to get one year of subscription to use Wikipedia for free!</p>
  </label>
</label>
/***
i want h3 heading caps only  first world can i add tailwind class according my needs customization ?

**Comment by saadeghi** [comment, marked as answer]
Yes you can add any Tailwind CSS utility class or any custom CSS to customize styles.  
Is there any problem?

**Reply by madfortech** [reply]
no problem
